doc/classes/SpringArm3D.xml

```diff
@@ -28,6 +28,12 @@
 				Returns the spring arm's current length.
 			</description>
 		</method>
+		<method name="is_colliding" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns whether the [SpringArm3D]'s ray or shape is intersecting with any colliders, and thus, if the spring arm's current length is less than its maximum length.
```

Comment:
```suggestion
				Returns whether the SpringArm3D's ray or shape is intersecting with any colliders, and thus, if the spring arm's current length is less than its maximum length.
```
As I mentioned in the suggestion, this isn't done anywhere else in the file so doing it here feels out of place

Reply:
Ah. Alright, let me roll it back^^'